pyanaconda/modules/payloads/payload/dnf/dnf_manager.py

```diff
@@ -191,6 +191,27 @@ def get_installation_size(self):
 
         return total_space
 
+    def get_download_size(self):
+        """Calculate the download size.
+
+        :return: a space required for packages
+        :rtype: an instance of Size
+        """
+        download_size = Size(0)
+
+        if self._base.transaction is None:
+            return Size(0)
```

Comment:
Moving this check to the top of the function would be better.

Reply:
Fixed.